addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -82,6 +82,9 @@ class ComposerTextInput extends Component {
         }
         if (this.composer.thread && this.composer.thread.model !== 'mail.channel') {
             if (this.composer.isLog) {
+                if (!this.composer.thread.hasWriteAccess) {
+                    return this.env._t("You cannot post on this Document...");
+                }
```

Comment:
You should add a new entry in `mail.pot` when introducing a new translatable entry in a stable version